src/utils/decorators/input-validation/__spec__.js

```diff
@@ -1213,11 +1213,13 @@ describe('InputValidation', () => {
 
       it('returns an info icon', () => {
         expect(instance.validationHTML[0].props.type).toEqual('info');
+        // TODO - 1007 - find not returning element though it is printed in the html()
         expect(wrapper.find('.common-input__icon.common-input__icon--info').exists()).toBeTruthy();
```

Comment:
not sure why it's not working - but you could do the same check we perform for other kinds of errors instead https://github.com/Sage/carbon/blob/4467a85bc8a72dfdce58b0f8d58c35b739735ceb/src/utils/decorators/input-validation/__spec__.js#L1114